traffic_ops/v3-client/deliveryservice.go

```diff
@@ -34,13 +34,25 @@ const (
 	// See Also: https://traffic-control-cdn.readthedocs.io/en/latest/api/v3/deliveryservices.html
 	API_DELIVERY_SERVICES = apiBase + "/deliveryservices"
 
+	// API_DELIVERY_SERVICES_V31 is the API path on which Traffic Ops serves Delivery Service
+	// information. More specific information is typically found on sub-paths of this.
+	// See Also: https://traffic-control-cdn.readthedocs.io/en/latest/api/v3/deliveryservices.html
+	API_DELIVERY_SERVICES_V31 = apiBaseV31 + "/deliveryservices"
```

Comment:
If we follow my aliasing comment above, I think we would just update `apiBase` to 3.1, and we wouldn't need the new 3.1-specific client methods below.